.github/workflows/docker.yml

```diff
@@ -0,0 +1,37 @@
+name: Docker Build and Sanity Check
+
+on:
+  push:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build Docker image
+        run: |
+          docker build -t wis2box-ui:test .
+
+      - name: Run Docker container
+        run: |
+          # TODO spin up a local WIS2BOX API and validate state of the UI
+          # Set the WIS2BOX_API_URL env var with -e to a dummy value so build passes.
```

Comment:
In the future I can add either a mocked API endpoint or the live demo server. For right now this test is mainly about testing the docker build more than the UI behavior after build. That is done easier in vitest directly. 

Once I write some playright tests for checking the UI I can use that on the dockerized version as well